src/Tabs.js

```diff
@@ -1,43 +1,74 @@
 import React from 'react';
 import cx from 'classnames';
+import idgen from './idgen';
 
 import Row from './Row';
 import Col from './Col';
 
 class Tabs extends React.Component {
+  componentDidMount() {
+    if (typeof $ !== 'undefined') {
```

Comment:
You don't need to explicitly throw an exception:

![image](https://cloud.githubusercontent.com/assets/7511692/15861263/693b60e4-2ca2-11e6-9b62-0d5c61d5660f.png)

Just believe that jQuery will be available on the page :)